packages/svelte-vscode/package.json

```diff
@@ -396,6 +396,16 @@
                 }
             }
         },
+        "configurationDefaults": {
+            "[svelte]": {
```

Comment:
I'm not 100% sure we want this because people can also use the prettier extension, if that ones installed I'd actually prefer that one was used assuming people have both prettier and the plugin installed.
The label thing is a super nice addition.

Reply:
What's the difference between the two? I've been using the svelte formatter as default although I have both installed. Mainly because setting prettier as the default formatter prompts me to change the default formatter when I try to format a svelte file in a project that has no prettier installed or prettier config.